### PR TITLE
[COMMUNITY] Update maturity model

### PR DESCRIPTION
**Pull request by pan3793**
### _Why are the changes needed?_

This PR proposes to update the Kyuubi project maturity model to reflect the current project status.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
I second to comments from our mentor, can you elaborate a bit more what&why can we update the model, especially add some solid evidence, not only says that `update .... reflect the current project status.` which is clear to everyone

File: MATURITY.md [hunk 1]
```diff
@@ -83,18 +83,18 @@ Mentors and community members are encouraged to contribute to this page and comm
...
+| CS40 | In Apache projects, vetoes are only valid for code commits. The person exercising the veto must justify it with a technical explanation, as per the Apache voting rules defined in CS30.                                  | :white_check_mark: The project has not used a veto at any point during incubation.                                                                                                                             |
+| CS50 | All "important" discussions happen asynchronously in written form on the project's main communications channel. Offline, face-to-face or private discussions that affect the project are also documented on that channel. | :white_check_mark: The project has been making important decisions on the project mailing lists. Minor decisions may occasionally happen during code reviews, which are also asynchronous and in written form. |
```
Comment (pan3793) [inline MATURITY.md:92]: For example:

Major features discussion:

- [[DISCUSS][KPIP-4] Support to submit batch job](https://lists.apache.org/thread/mfn69bhnrqgm67wcgvc08fbm0qwkfj99)
- [[KPIP-4] Support to submit batch job](https://github.com/apache/incubator-kyuubi/discussions/2304)

- [[DISCUSS][KPIP-5] Introduce Kyuubi Web UI](https://lists.apache.org/thread/5rx1sc7syny9bt8kpr6lnoyr4jmnhkkp)
- [[KPIP-5] Introduce Kyuubi Web UI](https://github.com/apache/incubator-kyuubi/discussions/2651)
- [[VOTE][KPIP-5] Introduce Kyuubi Web UI](https://lists.apache.org/thread/qk5ftp6jd0w37lv018mn4zx75d7m2n0w)

Deprecation discussion:

- [[DISCUSS] Spark 3.0 Support EOL Progress](https://lists.apache.org/thread/3mw8zz9r0tzg2dpj6717xy59610bhkp4)

File: MATURITY.md [hunk 2]
```diff
@@ -83,18 +83,18 @@ Mentors and community members are encouraged to contribute to this page and comm
...
+|------|------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| IN10 | The project is independent from any corporate or organizational influence.               | :white_check_mark: The project team gathers people from different companies (AsiaInfo, bilibili, China Mobile, eBay, iQIYI, NetEase, T3, Zhihu ...). No company or organization has significantly more influence than any other. We can note a growth of the contributions coming from different contributors.    |
```
Comment (pan3793) [inline MATURITY.md:98]: There are 12 PPMC members come from different companies, including AWS, eBay, Huawei, Sensors Data, NetEase, China Mobile, Ctrip, and T3
https://whimsy.apache.org/roster/ppmc/kyuubi

And the contributors is growing.
https://github.com/apache/incubator-kyuubi/graphs/contributors

File: MATURITY.md [hunk 3]
```diff
@@ -83,18 +83,18 @@ Mentors and community members are encouraged to contribute to this page and comm
...
+| CS30 | The project uses documented voting rules to build consensus when discussion is not sufficient.                                                                                                                            | :white_check_mark: The project uses the standard ASF voting rules.                                                                                                                                             |
+| CS40 | In Apache projects, vetoes are only valid for code commits. The person exercising the veto must justify it with a technical explanation, as per the Apache voting rules defined in CS30.                                  | :white_check_mark: The veto is used for code commits only, and each veto with a clear technical explanation.                                                                                                   |
```
Comment (pan3793) [inline MATURITY.md:91]: For instance: https://github.com/apache/incubator-kyuubi/pull/3486

**Comment by pan3793** [comment]
@yaooqinn @WillemJiang I supplied examples to explain why I think the project reached such a standard.

**Review by turboFei — APPROVED** [review]

**Review by cfmcgrady — APPROVED** [review]

**Review by cxzl25 — APPROVED** [review]

File: MATURITY.md [hunk 4]
```diff
@@ -83,18 +83,18 @@ Mentors and community members are encouraged to contribute to this page and comm
...
+|------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CS10 | The project maintains a public list of its contributors who have decision power. The project's PMC (Project Management Committee) consists of those contributors.                                                         | :white_check_mark: Committers and PPMC members are listed on the website’s community page, see [newcommitter](https://kyuubi.apache.org/become_committer.html)                                                 |
```
Comment (ulysses-you) [inline MATURITY.md:88]: as we are here, add one more link [new_ppmc] https://kyuubi.apache.org/become_ppmc_member.html

**Review by hddong — APPROVED** [review]

**Review by wForget — APPROVED** [review]

**Review by ulysses-you — APPROVED** [review]

**Review by WillemJiang — APPROVED** [review]

File: MATURITY.md [hunk 5]
```diff
@@ -83,18 +83,18 @@ Mentors and community members are encouraged to contribute to this page and comm
...
+| CS30 | The project uses documented voting rules to build consensus when discussion is not sufficient.                                                                                                                            | :white_check_mark: The project uses the standard ASF voting rules.                                                                                                                                                          |
+| CS40 | In Apache projects, vetoes are only valid for code commits. The person exercising the veto must justify it with a technical explanation, as per the Apache voting rules defined in CS30.                                  | :white_check_mark: The veto is used for code commits only, and each veto with a clear technical explanation.                                                                                                                |
```
Comment (yaooqinn) [inline MATURITY.md:91]: Let's change this sentence from passive voice to active.

We've used vetoes with clear technical explanations for code commits only.

**Review by yaooqinn — APPROVED** [review]
LGTM, only one typo

File: MATURITY.md (hunk 5 above)
Comment (pan3793) [reply MATURITY.md]: Thanks, updated

**Review by SteNicholas — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks all for your review and approval, the success of Kyuubi is inseparable from all of you! Merging to master.

**Pull request closed without merging**